packages/core/src/components/Tooltip/__stories__/tooltip.stories.scss

```diff
@@ -5,45 +5,61 @@
     gap: 16px;
   }
 
-  &_icon-wrapper {
+  &_title {
+    padding: 20px 0;
     display: flex;
-    padding: 6px;
-    gap: 8px;
-    align-items: center;
-    background-color: var(--sb-disabled-background-color);
-    cursor: pointer;
-    &:hover {
-      background-color: var(--sb-primary-background-hover-color);
-    }
   }
 
-  &_icon-box {
-    background-color: var(--sb-color-winter);
-    width: 54px;
-    height: 54px;
+  &_image {
+    padding: 100px 0;
     display: flex;
-    align-items: center;
-    justify-content: center;
   }
 
-  &_top {
-    padding: 0 64px 68px 0;
-    margin: 0 32px;
+  &_tabs {
+    ul,
+    li {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &_position-left {
+    position: relative;
+    left: -25%;
   }
 
-  &_bottom {
-    padding: 50px 0 0 0;
-    margin: 0 32px;
+  &_link {
+    color: var(--text-color-on-primary);
+    text-decoration: underline;
   }
 
-  &_left {
-    padding: 0 64px 8px 64px;
-    margin: 0 64px;
+  &_position-bottom {
+    position: relative;
+    top: 25%;
   }
 
-  &_right {
-    padding: 0 32px 8px 32px;
-    margin: 0 32px;
+  &_positions {
```

Comment:
The same values as before, just used nth child to save some classes to avoid confusion for users